Merl 586 finalize new faust website

TeresaGobble opened the pull request:
## Description

- Configured Sidebar
- Evaluated intro text on homepage (did not change this, as it is still accurate)
- Updated “features” on homepage to add Apollo
- Migrated Blog
- Removed lingering docs that don’t apply to new Faust
- Removed “edit this page” links from docs
- Updated twitter link to connect to the wpengine twitter 
- Fixed bug with who the article is by and when it was published (it was related to the production command in Atlas, which will run `npm run start` as the production branch, whereas this was the dev branch in the `config`'s script)

## Testing

Previews should show the site's changes and provide you with an opportunity to click through to ensure it is working. If not:

rom the root, cd into the `faustjs.org` folder.
In the command line, run `npm run build`
run `npm run dev'

theodesp commented:
Hey @TeresaGobble great job here. I've noticed the following issues with some pages:

1. Custom post types link points to a 404 page:
<img width="1308" alt="Screenshot 2022-11-02 at 11 37 21" src="https://user-images.githubusercontent.com/328805/199480300-75037014-c746-4986-85ed-565803b2f20b.png">
2. React hooks page point to old docs referencing GQty hooks. We should remove the link and the relevant docs from there:
<img width="1307" alt="Screenshot 2022-11-02 at 11 37 50" src="https://user-images.githubusercontent.com/328805/199480493-53e8ed76-9e83-4a51-883e-e3527f5b4180.png">

<img width="1301" alt="Screenshot 2022-11-02 at 11 38 47" src="https://user-images.githubusercontent.com/328805/199480560-802d20ce-d1f2-4392-93f2-bc1d2b1290bf.png">

3 Next.js, GraphQl and SSR/SSG links all point to 404 pages. We should either remove any links or change the links to a valid documentation page.

4. Both Tutorial and Getting started Links on the footer point to a 404 page:
<img width="1285" alt="Screenshot 2022-11-02 at 11 42 06" src="https://user-images.githubusercontent.com/328805/199481214-8a4c6b08-707f-43a4-a4b4-e05bd83df438.png">

blakewilson commented:
Nice work @TeresaGobble! let's also refine the menu a bit since with removing `Usage with Next.js` and just put everything at the top level. Something like:

```
- Getting Started
- Example Project Walkthrough
- Guides
- - Previews
- Reference
- - Seed Query
- - API Router
- - getNextServerSideProps
- - getNextStaticProps
- Companion WordPress Plugin
- - Settings Reference
- Telemetry
- Migration from old Faust
```

TeresaGobble commented:
@theodesp and @blakewilson , thank you so much for this detailed feedback! Will do!

TeresaGobble commented:
Thank you @theodesp and @blakewilson for your feedback! I have: 

Per Theo's feedback,
1. Deleted the Custom Post Types feature card from the front page
2. Deleted the React Hooks card from the front page
3. Deleted the Next.js, GraphQL and SSR/SSG cards from the front page
4. Updated the `getting started` link and deleted the `tutorial` link.

Per Blake's feedback, updated the sidebar menu tree.
![image](https://user-images.githubusercontent.com/50935135/199632825-f95d4c09-af1a-40f3-a00a-f5004e36fc6e.png)

Additionally, I added a seed query and nextjs card for the `getNextServerSideProps` function. Please let me know if you think other cards should be added! I added the example project to the footer as well.

![image](https://user-images.githubusercontent.com/50935135/199632938-d0f4634d-883b-4122-99e2-146c0c9e0348.png)

theodesp commented:
Hey Nice work here @TeresaGobble .

I just went through all the documentation site checking the content.  I found two small inconsistencies:

1. In the Using the API Router page, the code examples mention the old Faust code: `faustjs` instead of `faustwp`
<img width="1413" alt="Screenshot 2022-11-03 at 11 24 50" src="https://user-images.githubusercontent.com/328805/199708921-3e31a8ab-59d8-4cf2-b05d-fa30f3b67e8a.png">
2. The Migration Path page is empty. Maybe add a NOTE message for WORK IN PROGRESS.
<img width="1568" alt="Screenshot 2022-11-03 at 11 27 42" src="https://user-images.githubusercontent.com/328805/199709472-5bd7050c-c867-46bf-bcae-70326f651655.png">

josephfusco commented on internal/faustjs.org/docs/migrationPath/overview.mdx:
```diff
@@ -1,4 +1,7 @@
 ---
 slug: /migrationPath/overview
-title: Migrate from old version of Faust.js ( < v1.0.0 ) to Faust ( > v1.0.0 )
+title: Migration from Old Faust
```
```suggestion
title: Migration from Legacy Faust
```

josephfusco commented on internal/faustjs.org/sidebars.js:
```diff
@@ -91,48 +80,9 @@ module.exports = {
-        },
-      ],
+      type: 'doc',
+      label: 'Migration from old Faust',
```
```suggestion
      label: 'Migration from legacy Faust',
```

josephfusco reviewed (approved):
Great job on this!

Wondering if we should update "Old Faust" to "Legacy Faust"?
